install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -217,6 +217,7 @@ data:
   enable-envoy-config: "true"
   enable-gateway-api-secrets-sync: {{ .Values.gatewayAPI.secretsNamespace.sync | quote }}
   gateway-api-secrets-namespace: {{ .Values.gatewayAPI.secretsNamespace.name | quote }}
+  gateway-api-metrics-addr: ":{{ .Values.gatewayAPI.metricsPort }}"
```

Comment:
Please add this port to the metrics service as well, so they can be discovered via the ServiceMonitor infrastructure (though it seems that for the proxy, the infrastructure might also be incomplete).

https://github.com/cilium/cilium/blob/d914e6cd50965446fca04e71d4e199f3887c5872/install/kubernetes/cilium/templates/cilium-agent/service.yaml